src/chapters/08-working-with-data.md

```diff
@@ -194,9 +203,9 @@ wait  #qunit-banner.qunit-pass
 
 Now that we have things in place, let's do the fun part of removing *all* our hard-coded values from the model hook and actually fetch some data from the server!
 
-In a production app, the data that we'd fetch would most likely come from a remote API server. To avoid setting up an API server just for this tutorial, we will put some JSON data into the `public` folder instead. That way, we can still request these JSON data with regular HTTP requests &mdash; just like we would with a real API server  &mdash; but without having to write any server logic.
+In a production app, the data that we'd fetch would most likely come from a remote API server. To avoid setting up an API server just for this tutorial, we will put some JSON data into the `public` folder instead. That way, we can still request this JSON data with regular HTTP requests &mdash; just like we would with a real API server  &mdash; but without having to write any server logic.
```

Comment:
There are multiple JSON files, is this change correct?

Reply:
Depends on whom you ask. [OED](https://www.lexico.com/en/definition/data) says "data" is a mass noun and thus gets singular "this".

[APA](https://blog.apastyle.org/apastyle/2012/07/data-is-or-data-are.html), however, says to use "these data".

Dianne and I both made this face when we read "these data": 🤔

"This data" feels less formal too.

Reply:
The AP styleguide says it's singular in non-academic contexts. https://twitter.com/apstylebook/status/1111694537020370945?lang=en

That sounds about right to me.